CONTRIBUTING.md

```diff
@@ -39,7 +39,7 @@ bug, you may also create an associated issue. If possible, try to illustrate you
 *  Please create a Git branch for each pull request (PR). [Click here](https://help.github.com/en/articles/creating-a-pull-request-from-a-fork) for information on how to create a PR.
 *  Your contributed code should roughly follow the tidyverse [style guide](http://style.tidyverse.org). _Exceptions_ from this guide: function names in PascalCase (e.g. NewFunction()) and variable names as camelCase (e.g newVariable <- "hi").
 *  cchsflow uses [roxygen2](https://cran.r-project.org/package=roxygen2), with
-[Markdown syntax](https://cran.r-project.org/web/packages/roxygen2/vignettes/markdown.html),
+[Markdown syntax](https://bookdown.org/yihui/rmarkdown/markdown-syntax.html),
 for documentation.
 *  cchsflow uses [testthat](https://cran.r-project.org/package=testthat) (TBA). Adding tests to the PR makes it easier for me to merge your PR into the code base.
```

Comment:
Implemented now remove (TBA) and "easier for me " seems unprofessional suggest replacing with "makes merging the PR into the code base less prone to bugs"